### PR TITLE
Fix address book bootstrap failing with null `ip_address_v4`

### LINKED ISSUE #8844: Initial address book bootstrap fails for ip_address_v4
### Description

Importer is failing with logs:
`Caused by: org.postgresql.util.PSQLException: ERROR: null value in column "ip_address_v4" of relation "address_book_service_endpoint" violates not-null constraint`

### Steps to reproduce

1. Check importer logs on integration.

### Additional context

_No response_

### Hedera network

other

### Version

v0.110.0-SNAPSHOT

### Operating system

None

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:
This PR sets ip_address_v4 to an empty string.

This PR
* Sets  ip_address_v4 to an empty string instead of null

**Related issue(s)**:

Fixes #8844

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by steven-sheehy** [comment]
Needs a test that reproduces the issue.

**Review by jascks — APPROVED** [review]
LGTM

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/addressbook/AddressBookServiceImplTest.java [hunk 1]
```diff
@@ -751,6 +752,38 @@ void verifyAddressBookWithDeprecatedIpAndServiceEndpoints() throws UnknownHostEx
         assertEquals(addressBookEntries * numEndpointsPerNode, addressBookServiceEndpointRepository.count());
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"1.0.0.0", ""})
```
Comment (edwin-greene) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/addressbook/AddressBookServiceImplTest.java:756]: Would adding `null` as a value here be a good test to make sure it uses the `ByteString.EMPTY`?

Comment (mgoelswirlds) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/addressbook/AddressBookServiceImplTest.java:756]: I did try that, but that was getting converted to a valid IP here, so I removed it.

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by edwin-greene — APPROVED** [review]
Looks good

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**